src/server/auth/cmds/cmds.go

```diff
@@ -681,6 +681,39 @@ func GetEnterpriseRoleBindingCmd() *cobra.Command {
 	return cmdutil.CreateAlias(get, "auth get enterprise")
 }
 
+// RotateRootToken returns a cobra command that rotates the auth token for the Root User
+func RotateRootToken() *cobra.Command {
+	var rootToken string
+	rotateToken := &cobra.Command{
+		Use:   "{{alias}}",
+		Short: "Rotate the auth token for a user",
```

Comment:
nit: The usage should reflect that this only works for the root user and that it revokes any existing root token.